books/views.py

```diff
@@ -1,3 +1,19 @@
-from django.shortcuts import render
+from rest_framework import mixins, viewsets
+from books.models import Book
+from books.serializers import BookSerializer
+
+
+class BookViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    mixins.RetrieveModelMixin,
+    viewsets.GenericViewSet,
```

Comment:
In our task, we should realize all CRUD 
![image](https://user-images.githubusercontent.com/18147743/234017289-2ea26210-151b-4673-820d-0e4795058445.png)